### ISSUE TITLE
surfaces/integrations: build Segment integration

### ISSUE THREAD

**Issue by morsapaes** [MaterializeInc/materialize#25343]
### Feature request

Today, we integrate with Segment via the [webhook source](https://materialize.com/docs/ingest-data/segment/). Their team suggested we build an [Actions destination integration](https://github.com/segmentio/action-destinations/tree/main#action-destinations), so Materialize can be featured in the product. The first step is to develop and submit the integration to their `action-destinations` repo (TypeScript):

* [Create a new Destination Action](https://github.com/segmentio/action-destinations/blob/main/docs/create.md)
* [Submit a pull request](https://github.com/segmentio/action-destinations/blob/main/CONTRIBUTING.md#submit-a-pull-request)

Once this is done, the destination will be deployed to Segment and we can manage it via their Developer Portal. This is **not a priority**, and shouldn't take precedence over any integrations work that contributes to the Q1 roadmap.
